src/options.rs

```diff
@@ -108,3 +101,21 @@ impl TryFrom<&Args> for Options {
         })
     }
 }
+
+fn build_glob_set(glob_set: &Vec<String>) -> Result<Option<GlobSet>>{
+    let globset = if glob_set.is_empty() {
+        Ok(None)
+    } else {
+        let mut builder = GlobSetBuilder::new();
+        for glob_str in glob_set {
+            if glob_str.contains('/') {
```

Comment:
Maybe this should also match backslash on Windows, by also checking against https://doc.rust-lang.org/std/path/constant.MAIN_SEPARATOR.html as well as `/`. We could then have a unit test in this file that runs on Windows only and that checks that it matches only paths.

Reply:
So basically, we just need to check only against [std](https://doc.rust-lang.org/std/index.html)::[path](https://doc.rust-lang.org/std/path/index.html)::[MAIN_SEPARATOR](https://doc.rust-lang.org/std/path/constant.MAIN_SEPARATOR.html#), which will be then determined based on the execution platform, right?

Reply:
On Windows I think we should accept either. So 

`if glob_str.contains('/') || glob_str.contains(std::path::MAIN_SEPARATOR)`